### ISSUE TITLE
html is escaped.  How to unescape? 

### ISSUE THREAD

**Issue by vallard** [jormaechea/open-api-mocker#48]
Hi there!  I have my openapi yaml that has a snippet like this for the `response` portion:

```
responses:
        200:
          description: Success
          content:
            text/html:
              example: <form><input type="hidden" name="_csrf" value="5hchwoPz-Lx6J-naPVYQIwOrOgsM-lZXqzso" /></input><input type="hidden" name="transaction_id" value="CUUG6pIE" /><form/>
          headers:
            set-cookie:
              schema:
                type: string
                example: "session ID cookie"
```
When I get the response it prints out escapes the quotes: 

```
<form><input type=\"hidden\" name=\"_csrf\" value=\"5hchwoPz-Lx6J-naPVYQIwOrOgsM-lZXqzso\" /></input><input type=\"hidden\" name=\"transaction_id\" value=\"CUUG6pIE\" /><form/>
```

Perhaps I'm doing something wrong?  Thanks for this tool and for any support!

**Comment by jormaechea** [comment]
Hi @vallard. Sorry for the delay. Support for `text/*` content types has not been implemented yet.

If you want, you can submit a PR.

**Comment by jormaechea** [comment]
Feature released in `v1.9.0`
